test/files/Example.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
```

Comment:
This looks like a real-world-example. That in itself is not a problem, but it would be nice to have an example file that specifically covers different cases:

- enums
- nested messages
- maybe an imported input type, like `google.protobuf.Empty`?